sbin/suspend_program.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+node_list=$(echo $@ | sed "s/ /,/g")
```

Comment:
Unfortunately this has not fixed the issue i was seeing earlier. I am still seeing the same issue.

Reply:
Mysteriously the problem only comes in htc partition and not hpc 🤔